minutes/meeting-202001019-4.md

```diff
@@ -0,0 +1,129 @@
+# Minutes of Meeting 4, 2020-10-17, 16:00–18:30 UTC
+
+Attendees
+
+Present
```

Comment:
```suggestion
Present (IRC)
```

OR ditch this section, we probably don't need IRC vs. Video meeting attendees separately, a merged list or only the latter is probably good.